server/application/application.go

```
@@ -1196,6 +1196,16 @@ func (s *Server) PodLogs(q *application.ApplicationPodLogsQuery, ws application.
 		return err
 	}
 
+	// Temporarily, logs RBAC will be enforced only if an intermediate env var serverRBACLogEnforceEnable is defined and has a "true" value
+	// Otherwise, no RBAC enforcement for logs will take place (meaning, can-i request on a logs resource will result in "yes")
+	// In the future, logs RBAC will be always enforced and the parameter along with this check will be removed
+	logsRBACEnforceEnable, _ := s.settingsMgr.GetServerRBACLogEnforceEnable()
```

not ignore error here